base/applications/mspaint/selectionmodel.cpp

```diff
@@ -540,3 +540,25 @@ void SelectionModel::UnlockBitmap(HBITMAP hbmLocked)
 {
     m_hbmColor = hbmLocked;
 }
+
+void SelectionModel::ControlPlusMinus(BOOL bMinus)
```

Comment:
A more descriptive name would be better: something like `ZoomSelection(BOOL bZoomIn)` or similar.

Reply:
This feature actually stretches the selection.
Zooming is visual only.

Reply:
Ok then maybe call the function "StretchSelection"

Reply:
Done.